hedera-node/yahcli/src/test/java/com/hedera/services/yahcli/test/bdd/YahcliVerbs.java

```diff
@@ -110,4 +224,41 @@ public static void setDefaultWorkingDir(@NonNull final String workingDir) {
         requireNonNull(workingDir);
         DEFAULT_WORKING_DIR.set(requireNonNull(workingDir));
     }
+
+    /**
+     * Builds the absolute <b>filename</b> of a given key relative to Yahcli's default network. The
+     * resulting format should be an absolute path ending with {@code <default network name>/keys},
+     * e.g. {@code localhost/keys}.
+     * @param keyFile the file name of the key
+     * @return the path of the key file
+     */
+    public static String asYcDefaultNetworkKey(@NonNull final String keyFile) {
+        requireNonNull(keyFile);
+        return Path.of(absDefaultNetworkKeysDir(), keyFile).toString();
+    }
+
+    private static String absDefaultNetworkKeysDir() {
+        return Path.of(DEFAULT_WORKING_DIR.get(), TEST_NETWORK, "keys")
+                .toAbsolutePath()
+                .toString();
+    }
+
+    private static void extractAndAcceptValue(String output, Pattern pattern, LongConsumer consumer) {
+        final var m = pattern.matcher(output);
+        if (m.find()) {
+            consumer.accept(Long.parseLong(m.group(1)));
+        } else {
+            Assertions.fail("Expected '" + output + "' to contain '" + pattern.pattern() + "'");
+        }
+    }
+
+    /**
+     * Get Path of a resources file.
+     * @param resourceFileName the file name
+     * @return the Path
+     */
+    public static Path loadResourceFile(String resourceFileName) {
+        return Path.of(Objects.requireNonNull(YahcliVerbs.class.getClassLoader().getResource(resourceFileName))
+                .getPath());
```

Comment:
Does this also work when the module path is enabled? I couldn't get this to return the right value a few weeks ago